packages/components/app/styles/components/dropdown.scss

```diff
@@ -0,0 +1,410 @@
+  width: 2rem;
+
+  img {
+    border-radius: inherit;
+  }
+}
+
+.hds-dropdown-toggle-icon__chevron {
+  margin-left: 0.25rem;
+}
+
+// TOGGLE/BUTTON
+.hds-dropdown-toggle--with-button-component {
+  box-shadow: none; // we override this to remove the elevation style
+
+  .hds-button__icon {
+    margin-right: -6px; // we apply a negative margin to counter the padding-right of the button and reduce the visual space between the icon and the right border
+    margin-left: 0.5rem; // this overrides the rule `.hds-button__text + .hds-button__icon`
+  }
+}
+
+
+// LIST
+// UL ELEMENT
+// GOES INSIDE HDS::DISCLOSURE's :content block
+
+.hds-dropdown-list {
+  background-color: var(--token-color-surface-primary);
+  border-radius: 6px;
+  box-shadow: var(--token-surface-high-box-shadow);
+  list-style: none;
+  margin: 0;
+  max-width: 25rem;
+  min-width: 12.5rem;
+  padding: 0.25rem 0;
+  &.hds-dropdown-list--position-right {
+    position: absolute;
+    right: 0;
+    top: calc(100% + 0.25rem);
+    z-index: 2; // https://github.com/hashicorp/design-system/issues/114
+  }
+
+  &.hds-dropdown-list--position-left {
+    position: absolute;
+    left: 0;
+    top: calc(100% + 0.25rem);
+    z-index: 2; // https://github.com/hashicorp/design-system/issues/114
+  }
+}
+
+// LIST > LIST-ITEM
+// HDS::DROPDOWN::LIST-ITEM
+
+.hds-dropdown-list-item--title {
+  color: var(--token-color-foreground-strong);
+  font-family: var(--token-typography-body-100-font-family);
+  font-size: var(--token-typography-body-100-font-size);
+  font-weight: var(--token-typography-font-weight-semibold);
+  line-height: var(--token-typography-body-100-line-height);
+  padding: 0.625rem 1rem 0.25rem;
+}
+
+.hds-dropdown-list-item--generic {
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.hds-dropdown-list-item--description {
+  color: var(--token-color-foreground-faint);
+  font-family: var(--token-typography-body-100-font-family);
+  font-size: var(--token-typography-body-100-font-size);
+  font-weight: var(--token-typography-font-weight-regular);
+  line-height: var(--token-typography-body-100-line-height);
+  padding: 0.125rem 1rem 0.25rem;
+}
+
+.hds-dropdown-list-item--separator {
+  position: relative;
+  height: 0.25rem;
+  width: 100%;
+
+  &::before {
+    position: absolute;
+    right: 6px;
+    left: 6px;
+    bottom: 0;
+    border-bottom: 1px solid var(--token-color-border-primary);
+    content: '';
+  }
+}
+
+.hds-dropdown-list-item__copy-item-title {
+  color: var(--token-color-foreground-faint);
+  font-family: var(--token-typography-body-100-font-family);
+  font-size: var(--token-typography-body-100-font-size); // 13
+  font-weight: var(--token-typography-font-weight-semibold);
+  line-height: var(--token-typography-body-100-line-height); // 18
+  padding: 0.125rem 0 0.25rem;
+}
+
+.hds-dropdown-list-item--copy-item {
+  box-sizing: border-box;
+  padding: 0.625rem 1rem 0.75rem;
+  width: 100%;
+
+  button {
+    @include hds-focus-ring-basic();
```

Comment:
[chore/consistency] move this declaration just before the other states

Reply:
can you explain more please? It makes maintenance harder to have these (`@include`, `@mixin` etc) not at the top.

Reply:
in all the other cases where these focus-ring mixins are used, they're used just before the other states, so it's easy to find the "focus" state in code (it's always in the block of states), and it's easy to see if that's applied and how. See for example in the Breadcrumb ([here](https://github.com/hashicorp/design-system/blob/d15ee400dbc4827a98de8a054499b358f34b2103/packages/components/app/styles/components/breadcrumb.scss#L78-L97) and [here](https://github.com/hashicorp/design-system/blob/d15ee400dbc4827a98de8a054499b358f34b2103/packages/components/app/styles/components/breadcrumb.scss#L156-L174))